src/dataexplorer/dataexplorer-application/src/components/Navigation/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -14,26 +9,23 @@ const Breadcrumbs = () => {
     return (
         <>
             <div className="breadcrumbs-container">
-                <Breadcrumb>
+                <Breadcrumb separator=">">
                     <Breadcrumb.Item>
                         <Link to="/">
-                            <HomeOutlined />
                             <span className="breadcrumbs-item">Hjem</span>
                         </Link>
                     </Breadcrumb.Item>
                     {name !== '' && (
                         <Breadcrumb.Item>
                             <Link to="/pasient">
-                                <UserOutlined />
                                 <span className="breadcrumbs-item">{name}</span>
                             </Link>
                         </Breadcrumb.Item>
                     )}
                     {schemaNumber !== '' && (
                         <Breadcrumb.Item>
-                            <FileTextOutlined className="bread-icon" />
                             <span className="breadcrumbs-item">
-                                Skjema - {schemaNumber}
+                                <b>Skjema {schemaNumber}</b>
```

Comment:
The <b> tag is not recommended for styling. Suggest to use <strong> or font-weight instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Reply:
100% agree. This wasn't added now, so unsure why GIT says it was 🍶